### ISSUE TITLE
Example Readmes

### ISSUE THREAD

**Issue by theIsaacLim** [dguo/make-a-readme#14]
I've come up with a list of example readmes. I'll add these in a new section of the website along with a short description of each one and what you can learn from it.
Here are the READMES:
https://github.com/sindresorhus/pageres
https://github.com/akashnimare/foco
https://github.com/ai/size-limit
https://github.com/sr6033/lterm
https://github.com/amitmerchant1990/electron-markdownify

**Comment by davidak** [comment]
I wanted to suggest you to open a new issue and look for some examples, but you already did. Perfect!

They are all JavaScript. I think they should come from different communities and languages.

5 examples might be good. I suggest these popular languages:

1. JavaScript
2. Java
3. Python
4. C or C++
5. one non-code example like dataset or website

Source: https://hackernoon.com/top-3-most-popular-programming-languages-in-2018-and-their-annual-salaries-51b4a7354e06

We should have different projects like a web application (probably JS), desktop app with GUI (Java or C?) and command line program (Python, C?). Maybe also a library.

For more examples, we can link to https://github.com/matiassingers/awesome-readme

**Comment by dguo** [comment]
Yep, I agree with adding some diversity since READMEs are language and platform agnostic.

**Comment by theIsaacLim** [comment]
@davidak I've selected a few-could you look through them?

Library, Python
https://github.com/keras-team/keras
CLI, Python
https://github.com/socialcopsdev/camelot
Dataset
https://github.com/googlecreativelab/quickdraw-dataset
Desktop app, JS
https://github.com/amitmerchant1990/electron-markdownify
Web library, JS
https://github.com/nodejs/node

**Comment by davidak** [comment]
@theIsaacLim they look good. (i like to see that google released the drawings!)

But i think we should define first what an example readme should look like. I think they should basically follow our suggestion. Then we can check if they match. Especially i want to discuss the sections. https://github.com/dguo/make-a-readme/issues/17
